gptel.el

```diff
@@ -253,18 +253,14 @@ By default, `gptel-host' is used as HOST and \"apikey\" as USER."
         secret)
     (user-error "No `gptel-api-key' found in the auth source")))
 
-;; FIXME Should we utf-8 encode the api-key here? 
+;; FIXME Should we utf-8 encode the api-key here?
 (defun gptel--api-key ()
   "Get api key from `gptel-api-key'."
   (pcase gptel-api-key
     ((pred stringp) gptel-api-key)
     ((pred functionp) (funcall gptel-api-key))
     (_ (error "`gptel-api-key' is not set"))))
 
-(defsubst gptel--numberize (val)
```

Comment:
Please add `gptel--numberize` back in, as I need this elsewhere in the code.  The rest of the changes look good!

Reply:
Hello, it seems that I have already removed all the places where `gptel--numberize` was used. Is this function no longer being used in the entire project?

Reply:
Yes, it is no longer being used. I will need to use it again in the future though, so it will be better to just leave it in.

Reply:
OK, I have added this function back ;)